packages/app/src/app/components/CreateSandbox/Import/Import.tsx

```diff
@@ -54,19 +68,53 @@ type ImportProps = {
 export const Import: React.FC<ImportProps> = ({ onRepoSelect }) => {
   const { hasLogIn, activeTeam } = useAppState();
   const importAndRedirect = useImportAndRedirect();
+  const {
+    hasActiveSubscription,
+    hasPastOrActiveSubscription,
+  } = useSubscription();
+  const { isTeamAdmin, isPersonalSpace } = useWorkspaceAuthorization();
+  const checkout = useGetCheckoutURL({
+    team_id: isTeamAdmin || isPersonalSpace ? activeTeam : undefined,
+    success_path: dashboard.recent(activeTeam),
+    cancel_path: dashboard.recent(activeTeam),
+  });
+
+  const checkoutURL = React.useMemo(() => {
+    if (isTeamAdmin || isPersonalSpace) {
+      if (checkout.state === 'READY') {
+        return checkout.url;
+      }
+      return '/pro';
+    }
 
+    return hasPastOrActiveSubscription
```

Comment:
I think `isEligibleForTrial` might be a better flag to use here? since it doesn't create any confusion with hasPastOrActive vs hasActive subscription flags